ReactiveCocoa/Swift/Action.swift

```diff
@@ -191,13 +191,7 @@ public final class CocoaAction: NSObject {
 	/// object given to execute(), if it can be downcast successfully, or nil
 	/// otherwise.
 	public convenience init<Input: AnyObject, Output, Error>(_ action: Action<Input?, Output, Error>) {
-		self.init(action, { input in
-			if let input: AnyObject = input {
-				return input as? Input
-			} else {
-				return nil
-			}
-		})
+		self.init(action, { $0 as? Input })
```

Comment:
I'v read [the type casting document](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/TypeCasting.html), but there was no description about taking optional values. :disappointed: 

But like the case in [ObjectiveCBridging.swift](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/f7611e4747816e9828ab8b48674a1e531c64c4d4/ReactiveCocoa/Swift/ObjectiveCBridging.swift#L149), casting operators can handle optional values as its input anyway. Thank you for the discussion @NachoSoto @jspahrsummers :bow: